blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,291 @@
+
+**We are planning to offer Wing Console for production**. This means that users will be able to
+connect to a deployed system and operate their production systems through Wing Console. 
+
+We believe this addresses a major pain when operating cloud applications. Today, we manage
+applications "from the bottom", i.e. from the resources up. I go to my cloud provider's
+administration console and what I see is a bunch of low-level resources - queues or buckets or
+functions. I don't see my application. I oftentimes also have to juggle to separate administrative
+tools for services that are outside of my provider's platform.
+
+Going back to my favorite Lego analogy: Think about a cloud application as this beautiful Lego
+castle. I've finished building my castle, and now I want to open the main gate. I log into the
+provider's console and I see a menu of Lego block types: all the reds, all the 4 piece blues, all
+the flat surfaces, etc. My gate is built out of reds and blues and greens and a bunch of cables. But
+I can't see a gate. I just see the building blocks from which it's constructed. How would I know
+which blue piece to move and where in order to open the gate?
+
+This is the experience we have today when we operate applications in the cloud. We get an alarm that
+there are some errors from some Lambda function. How do we know which part of the system is
+affected? Which team owns this part?
+
+The solution, today, is dashboards. I have to manually craft these views to represent the mental
+model of my system. I take the red and green and blue pieces and arrange them together under the
+title "Gate", and now if there's an error, I can tell where it belongs.
+
+The operational experience of Wing Console is based on Wing's preflight structure of the system.
+Every resource or event can be *traced back* to the logical unit it belongs to, based on your source
+code. This is basically the purpose of stack traces in traditional languages - an exception is
+thrown and the stack trace helps me trace back from low-level to high-level so I can diagnose the
+problem.
+
+
+### Fully customizable below the abstraction
+
+We are not naive to think that we can create an abstraction that will cover all use cases and
+possible setups. In fact, Wing is designed from the ground up to *celebrate cloud choice and
+diversity* and includes multiple levels of extensibility and customizability.
+
+One dimension of extensibility is support for any resource from the IAC ecosystem of the target as
+first-class citizens. For example, if you compile your code to a Terraform target, you can use any
+resource in the [Terraform Registry](https://registry.terraform.io). If you compile to AWS
+CDK/CloudFormation, you can use any [AWS CDK construct](https://docs.aws.amazon.com/cdk/api/v2/) in
+your code (L1s, L2s and any construct library from the [Construct Hub](https://constructs.dev/)).
```

Comment:
I would remove the L1, L2 reference, I assume that the reader that will know about the CDK will know about L1,L2 and the construct hub
```suggestion
your code.
```